brightchain-lib/src/lib/blocks/handle.ts

```
@@ -5,221 +5,256 @@ import {
       canPersist,
     );
-    block._path = path;
-
-    // Validate if we calculated the checksum ourselves
-    if (!checksum) {
-      await block.validateAsync();
-    }
-
-    return block;
   }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)